modules/fts/pages/fts-introduction.adoc

```diff
@@ -0,0 +1,124 @@
+
+= Introduction to Full Text Search
+
+== Full Text
+
+Provided by the xref:services-and-indexes/services/search-service.adoc[Search Service], this is a specially purposed index, which contains targets derived from the textual contents of documents within one or more specified keyspaces.
+
+Text-matches of different degrees of exactitude can be searched for.
```

Comment:
Again, "different degrees of exactitude" doesn't sound natural in American English.  I might say something like, "In addition to exact matches, the full-text index can perform fuzzy matching of terms.

Reply:
Update: Use the recommendation by Marty: "In addition to exact matches, the full-text index can perform fuzzy matching of terms"